src/manifest-dev.js

```diff
@@ -29,6 +29,6 @@ module.exports = {
     },
   ],
   manifest_version: 2,
-  content_security_policy: "script-src 'self'; object-src 'self'",
+  content_security_policy: "script-src 'self' 'unsafe-eval'; object-src 'self'",
```

Comment:
I removed on previous commit, but caused development stop working, so adding it back.

Reply:
I removed on previous commit, but caused development stop working, so adding it back. I assume development loading files separately.